docker-compose-configuration/rds-local-apps/rds-local/config/shiro-roles.ini

```diff
@@ -0,0 +1,6 @@
+[roles]
```

Comment:
why do we have a specific role file here? Unless we added additional permissions this should not be here.

Reply:
removed.